data-raw/process_data_sets_maggie.R

```diff
@@ -0,0 +1,72 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+
+
+
+# nba-carmelo---------------------------------------------------------------------
+#Tame Data Principle 1: naming conventions
+nba_carmelo <- read.csv("data-raw/nba-carmelo/nba_elo.csv")
+colnames(nba_carmelo) <- colnames(nba_carmelo) %>%
+  tolower() %>%
+  str_replace_all(" ", "_")
+  #converting variable names to snake_case
```

Comment:
Good job following the principles. When you're done however, please erase these comments.

Reply:
Comments enumerating the principles I mean; you should still comment what it is you're doing, of course!